styles.css

```diff
@@ -10,3 +10,19 @@
 .quizgame-scores {
     text-align: center;
 }
+.floating-button-container {
+    background: #FFF;
+    border: 0.5rem solid #FFF;
+}
+.floating-game-canvas {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 999999;
+}
+[role="main"] {
```

Comment:
This is to disable selection of the title and actual canvas, which is a reoccurring problem for iOS devices.